.circleci/config.yml

```diff
@@ -25,6 +25,7 @@ build-template: &build-template
         name: "Precommit and Coverage Report"
         command: |
           make ci
+          bash <(curl -s https://codecov.io/bash)
```

Comment:
I would much prefer to use a versioned build step to piping a downloaded bash script: https://circleci.com/docs/2.0/code-coverage/#codecov

Reply:
@MrAlias I changed this to use the codecov orb.  Take another look.